### ISSUE TITLE
go-ole checksum mismatch

### ISSUE THREAD

**Issue by jrick** [marcopeereboom/sbox#2]
After #1:

```
$ go mod tidy
verifying github.com/go-ole/go-ole@v1.2.2/go.mod: checksum mismatch
        downloaded: h1:pnvuG7BrDMZ8ifMurTQmxwhQM/odqm9sSqNe5BUI7v4=
        go.sum:     h1:7FAglXiTm7HKlQRDeOQ6ZNUHidzCWXuZWq/1dTyBNF8=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

**Comment by jrick** [comment]
@marcopeereboom, this `go.sum` file is completely screwy.  There are several modules, appearing as different versions, with the exact same hash.  wtf?

**Comment by jrick** [comment]
Fixed by 21078465.
